File: Python/aparatoGolgi.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from typing import List

class aparatoGolgi(object):
	def getProteinaGenerada(self) -> int:
		return self.___proteinaGenerada

	def setProteinaGenerada(self, aProteinaGenerada : int):
		self.___proteinaGenerada = aProteinaGenerada

	def getLipidosGenerados(self):
		return self.___lipidosGenerados

	def setLipidosGenerados(self, aLipidosGenerados):
		self.___lipidosGenerados = aLipidosGenerados

	def __init__(self):
		self.___proteinaGenerada : int = None
		self.___lipidosGenerados = None
		"""# @AssociationKind Composition"""

File: C#/aparatoGolgi.cs
using System;

public class AparatoGolgi {
	private int proteinaGenerada;
	private String lipidosGenerados;

	public int GetProteinaGenerada() {
		return this.proteinaGenerada;
	}
	public void SetProteinaGenerada(ref int proteinaGenerada) {
		this.proteinaGenerada = proteinaGenerada;
	}
	public void GetLipidosGenerados() {
		return this.lipidosGenerados;
	}
	public void SetLipidosGenerados(ref object lipidosGenerados) {
		this.lipidosGenerados = lipidosGenerados;
	}

	private Celula celula;

}
